news/2020-04-13-osu-spotlights-new-team-and-league-system.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: "osu! Spotlights: Recruiting New Team"
+date: 2020-04-13 08:00:00 +0000
+---
+
+Following our latest Spotlight season released very recently, we are underway reworking osu! Spotlights from the ground up, starting with recruiting for a brand new team and adding a whole new league system in future seasons.
+
+![](/wiki/shared/news/banners/beatmap-spotlights.jpg)
+
+Since March 2017, osu! Spotlights have worked in their current shape and form. While many mappers and players already love the idea of beatmaps being put on a pedestal, we want to expand on the idea by giving the curation and participation in osu! Spotlights a way better reason. While recommending beatmaps to you, a new competitive and seasonal league closely attached to the chosen beatmaps is going to be added. With this, you and your friends will have even more reason to delve deep into the best creations osu! has to offer.
+
+To accomplish this, a revamp of the Spotlights Team will be necessary. For that, we are now recruiting for community members willing and capable of selecting the best beatmaps created for all kinds of players so we can recommend them to everybody.
+
+**If you want to be part of the new Spotlights Team and help shape this from the very beginning, [you can hand in your application now!](https://spotlights.team/app)** We are looking for a diverse mix of members, including but not limited to renown mappers, modders, players, mentors, tournament organizers and mappool creators.
+
+Regarding the league, the exact details of how this concept will work are still in the making, but here are some aspects we *plan* to do:
+
+- **We plan to host weekly timeshift lobbies** using osu!lazer to have people compete actively on a portion of the beatmaps we have selected for the Spotlights. If you have never heard of timeshift multiplayer lobbies before, [you should check them out!](https://github.com/ppy/osu/releases)!
+- **We will release the Spotlights little by little instead of a whole seasonal release**. Therefore, **beatmaps being played in the lobby will change every week**. Those beatmaps will come from a portion of the whole Beatmap Spotlights Season.
```

Comment:
```suggestion
- **Spotlights will be released incrementally in a weekly fashion instead of all at once in big seasonal releases**. As above, this means that **beatmaps being played in the Spotlights league lobbies will change every week**.
```